docs/topics/assembly_creating-a-basic-spring-boot-application.adoc

```diff
@@ -0,0 +1,16 @@
+
+[id="creating-a-basic-spring-boot-application_{context}"]
+= Creating a Basic {runtime} Application
```

Comment:
`:parameter-runtime-name:` is not necessary because we already use `:runtime:`. When we need to list all attributes used in modules, let's do that, but right now, it's not necessary IMO, and it would make automatic replacement later more difficult.